### PR TITLE
Fix issue - Updating Profile Picture/Display Name doesn't work sometimes. #1207 

### PR DESCRIPTION
**Pull request by anna20210512**
### Pull Request Checklist

* [x] Pull request is based on the develop branch
* [x] Pull request contains a changelog file in ./changelog.d. See https://github.com/matrix-org/matrix-ios-sdk/blob/develop/CONTRIBUTING.md#changelog
* [x] Pull request includes a [sign off](https://github.com/matrix-org/matrix-ios-sdk/blob/develop/CONTRIBUTING.md#sign-off)

### CODE REVIEW

**Comment by anna20210512** [comment]
related issue - https://github.com/matrix-org/matrix-ios-sdk/issues/1207

File: changelog.d/1207.bugfix [hunk 1]
```diff
@@ -0,0 +1 @@
+MXUser.m: Add a property `latestUpdateTS` to update the user's avatar and displayname only when event.originServerTs > latestUpdateTS. Fixes issue - https://github.com/matrix-org/matrix-ios-sdk/issues/1207. Contributed by Anna.
```
Comment (ismailgulek) [inline changelog.d/1207.bugfix:1]:
```suggestion
MXUser.m: Add a property `latestUpdateTS` to update the user's avatar and displayname only when event.originServerTs > latestUpdateTS. Contributed by Anna.
```

Comment (ismailgulek) [reply changelog.d/1207.bugfix:1]: The issue reference added automatically.

File: MatrixSDK/Data/MXUser.m [hunk 2]
```diff
@@ -66,15 +68,17 @@ - (void)updateWithRoomMemberEvent:(MXEvent*)roomMemberEvent roomMember:(MXRoomMe
...
             self.displayname = [roomMember.displayname copy];
         }
-        if (roomMember.avatarUrl)
+        if (roomMember.avatarUrl && roomMemberEvent.originServerTs > latestUpdateTS)
         {
             self.avatarUrl = [roomMember.avatarUrl copy];
         }
-        
+        if (roomMemberEvent.originServerTs > latestUpdateTS) {
+            latestUpdateTS = roomMemberEvent.originServerTs;
```
Comment (ismailgulek) [inline MatrixSDK/Data/MXUser.m:80]: Can you do the update only when we update the `displayname` and `avatarUrl`?

File: MatrixSDK/Data/MXUser.m [hunk 3]
```diff
@@ -112,7 +116,9 @@ - (void)updateWithPresenceEvent:(MXEvent*)presenceEvent inMatrixSession:(MXSessi
             self.avatarUrl = nil;
         }
     }
-    
+    if (presenceEvent.originServerTs > latestUpdateTS) {
+        latestUpdateTS = presenceEvent.originServerTs;
```
Comment (ismailgulek) [inline MatrixSDK/Data/MXUser.m:120]: Can you do the update only when we update the `avatarUrl` from `presenceContent`?

**Comment by ismailgulek** [comment]
@anna20210512, thanks for fixing the issue.

**Pull request merged**